build/ci/Makefile

```diff
@@ -132,6 +132,26 @@ ci-e2e: vault-gke-creds
 		cloud-on-k8s-ci-e2e \
 		bash -c "make -C operators ci-e2e GKE_MACHINE_TYPE=n1-standard-8"
 
+ci-e2e-rc: vault-gke-creds
```

Comment:
What is `rc`?
Maybe worth adding a comment on top of the target to explain what it does (or when it's supposed to be used), and what's the difference with the other `ci-e2e` target.

Reply:
> What is rc?

Short from `release candidate`

> Maybe worth adding a comment on top of the target 

Sure, will do. Probably missed it

Reply:
Fixed

Reply:
`against provided ECK image `  vs. `ci-e2e-rc`. This seems to work beyond the concept of a release candidate.
Maybe `ci-e2e-custom-image`?